README.md

```diff
@@ -31,7 +31,16 @@ The `POSTGRES_DB_NAME` environment variable is mandatory, as the different servi
 
 If you don't want to spin up the containers, you'll need to locally run the required services (database, Kafka, etc.). You may need to add or remove the `managed` tag in the `${SERVICE}_tests.sh`.
 
-> **_NOTE:_**  Don't forget to set up `PATH` environment variable correctly when tests are run outside containers. That environment variable needs to also contain directory with tested executable files (for example `insights-results-aggregator-cleaner` etc.)
+> **_NOTE:_**  Don't forget to set up `PATH` environment variable correctly when runnig tests both within and outside containers. That environment variable needs to contain the path with tested executable files (for example `insights-results-aggregator-cleaner` etc.).
+
+When running the tests within the docker containers, it is necessary to have said executable under the `$VIRTUAL_ENV_BIN` folder with the right permissions. For example, after compiling `insights-results-aggregator`:
```

Comment:
> it is necessary to have said executable

"said"? I don; t get it

Reply:
is it "safe" (?)

Reply:
nono, it is "said". "Said executable", as in the previously mentioned